### PR TITLE
Update property with correct key naming

### PR DESCRIPTION
**Pull request by AndrewCS149**
The property "AzureSecurityCenter/Enable" has been updated to "DefenderForCloud/Enable"

### CODE REVIEW

**Comment by AndrewCS149** [comment]
#sign-off

File: test/snippets/expected/Automanage Create Custom Profile.snippetresult.json [hunk 1]
```diff
@@ -27,7 +27,7 @@
 					"Backup/RetentionPolicy/DailySchedule/RetentionDuration/DurationType": "Days",
 					"WindowsAdminCenter/Enable": false,
 					"VMInsights/Enable": true,
-					"AzureSecurityCenter/Enable": true,
+                    "DefenderForCloud/Enable": true,
```
Comment (StephenWeatherford) [inline test/snippets/expected/Automanage Create Custom Profile.snippetresult.json:30]: use tabs

**Review by StephenWeatherford — CHANGES_REQUESTED** [review]
whitespace issue

File: test/snippets/expected/Automanage Create Custom Profile.snippetresult.json (hunk 1 above)
Comment (AndrewCS149) [reply test/snippets/expected/Automanage Create Custom Profile.snippetresult.json:30]: Im no sure exactly how to fix this, everything looks good on my end: 
image

**Pull request merged**